lib/generators/scenic/materializable.rb

```diff
@@ -10,12 +10,21 @@ module Materializable
           required: false,
           desc: "Makes the view materialized",
           default: false
+        class_option :materialized_no_data,
```

Comment:
I think I'd prefer these to be separate options: `--materialized` and `--no-data`. I could see `--no-data` possibly implying `--materialized` but I'm not certain.